catkin_home/src/navigation/robot_description/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(robot_description)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  rospy
+)
+
+catkin_package()
+
+find_package(roslaunch)

```

Comment:
Are you sure this `find_package(roslaunch)` is needed? It seems that to use launch files this is not required.